src/App.js

```diff
@@ -7,7 +10,15 @@ class App extends React.Component {
   render() {
     return (
       <div>
-        <h2>Welcome to your Todo App!</h2>
+        <div>
```

Comment:
I would advise against adding a lot of div tags in your app.js. This is where you want to bring in all your components, so if you really need those divs, why not add them to your components?